configure

```diff
@@ -77,15 +77,16 @@ freshen_chef_repo
 pushd chef_repo
 wd="$(pwd)"
 
+if (test -n "$CHEF_MAGE_PROFILE" && test -z "$MAGE_KEY") || (test -z "$CHEF_MAGE_PROFILE" && test -n "$MAGE_KEY") ; then 
```

Comment:
Tempted to disagree here citing #5 as a counterpoint. We want to move these scripts to `sh` compatible in order to use them on mac without having to brew install bash.

Reply:
This test can be done with one test command rather than many.
```suggestion
if test \( -n "$CHEF_MAGE_PROFILE" -a test -z "$MAGE_KEY" \) -o \( test -z "$CHEF_MAGE_PROFILE" -a test -n "$MAGE_KEY" \); then 
```
I think that escaping the grouping parentheses rather than quoting them to avoid shell interpretation is easier to read but that's flexible.